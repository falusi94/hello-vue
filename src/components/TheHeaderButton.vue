<template>
  <BaseButton
    class="header-button"
    @click="$emit('click')"
    :title="title"
  />
</template>

<script>
export default {
  name: 'TheHeaderButton',
  props: [
    'title',
  ],
};
</script>

<style lang="scss" scoped>
  .header-button {
    &:hover {
      transition-delay: 0.1s;
      transition-duration: 0.2s;
      border-bottom: 2px solid #FFF;
    }
  }
</style>
