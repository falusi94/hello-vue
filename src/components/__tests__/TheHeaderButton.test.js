import { shallowMount } from '@vue/test-utils';
import TheHeaderButton from '../TheHeaderButton.vue';

describe('TheHeaderButton.vue', () => {
  it('renders properly', () => {
    const wrapper = shallowMount(TheHeaderButton);
    expect(wrapper).toMatchSnapshot();
  });
});
