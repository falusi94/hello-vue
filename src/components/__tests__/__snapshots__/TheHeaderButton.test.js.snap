// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TheHeaderButton.vue renders properly 1`] = `<basebutton class="header-button"></basebutton>`;
